src/main/java/com/jaenyeong/mission01/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,34 @@
+package com.jaenyeong.mission01.calculator;
+
+import java.util.List;
+
+import static com.jaenyeong.mission01.calculator.UI.inputExpression;
+import static com.jaenyeong.mission01.calculator.UI.printResultNumber;
+
+public class StringAddCalculator {
+    public static final int ONLY_ONE = 1;
+    public static final int FIRST = 0;
+
+    public static void calculate() {
+        final String inputExpression = inputExpression();
+
+        final int resultSum = splitAndSum(inputExpression);
+
+        printResultNumber(resultSum);
+    }
+
+    protected static int splitAndSum(final String inputExpression) {
+        final List<Integer> inputNumbers = Parser.parseToNumbers(inputExpression);
+
+        if (inputNumbers.size() == ONLY_ONE) {
+            return inputNumbers.get(FIRST);
+        }
+
+        return getSum(inputNumbers);
```

Comment:
```suggestion
        return getSum(inputNumbers);
```

해도 동일한 결과를 리턴 할 것 같아요. 어떻게 생각하시나요?